elasticsearch_dsl/utils.py

```diff
@@ -365,6 +365,7 @@ def __setattr__(self, name: str, value: Any) -> None:
 
     def _setattr(self, name: str, value: Any) -> None:
         # if this attribute has special type assigned to it...
+        name = AttrDict.RESERVED.get(name, name)
```

Comment:
Another bug that I noticed. The `AttrDict` handles `from_` and `from` correctly, but the `DslBase` class overrides this behavior and also needs to understand reserved names.